charts/testkube-api/templates/testworkflows/builtin-templates/v1/jmeter.yaml

```diff
@@ -27,6 +27,6 @@ spec:
   steps:
   - name: Run JMeter tests
     run:
-      image: {{`"justb4/jmeter:{{ config.version }}"`}}
+      image: {{`"alpine/jmeter:{{ config.version }}"`}}
```

Comment:
These are not inter-changeable images, so it should be at least a new template version